smrf/envphys/thermal/topotherm.py

```diff
@@ -0,0 +1,117 @@
+import numpy as np
+
+from smrf.envphys.constants import (FREEZE, GRAVITY, MOL_AIR, SEA_LEVEL,
+                                    STD_LAPSE, STD_LAPSE_M, STEF_BOLTZ, RGAS)
+from smrf.envphys.vapor_pressure import sati
+
+
+def brutsaert(ta, l, ea, z, pa):
+    """
+    Calculate atmosphere emissivity from Brutsaert (1975):cite:`Brutsaert:1975`
+
+    Args:
+        ta: air temp (K)
+        l: temperature lapse rate (deg/m)
+        ea: vapor pressure (Pa)
+        z: elevation (z)
+        pa: air pressure (Pa)
+
+    Returns:
+        atmosphericy emissivity
+
+    20151027 Scott Havens
+    """
+
+    t_prime = ta - (l * z)
+    rh = ea / sati(ta)
+    rh[rh > 1] = 1
+
+    e_prime = (rh * sati(t_prime))/100.0
+
+    air_emiss = (1.24*np.power(e_prime/t_prime, 1./7.0))*pa/SEA_LEVEL
```

Comment:
Could use some linting ....

Reply:
some linting and more verbose variable names